docs/topics/BrowserPlugin.adoc

```diff
@@ -66,9 +66,10 @@ The KeePassXC-Browser extension lets you automatically populate the entries from
 
 2. Open your web browser. The KeePassXC-Browser extension icon in your browser window will change based on its connection state. The figure below shows the different states.
 +
-*(A)* KeePassXC is not running or is disconnected +
-*(B)* Connected to KeePassXC, but database is locked +
-*\(C)* Connected to KeePassXC and ready to use
+*(A)* KeePassXC is not running or is disconnected. +
+*(B)* KeePassXC is running but current database is not connected to it. +

```

Comment:
I do not understand what it means that KeePassXC is not connected to the current database.

Reply:
KeePassXC _Browser Extension_ is not connected to the current database